general/okina.hpp

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+
+#ifndef MFEM_OKINA_HPP
+#define MFEM_OKINA_HPP
+
+#include "../config/config.hpp"
+#include "../general/error.hpp"
+
+#include <cmath>
+#include <cstring>
+#include <iostream>
+
+#include "cuda.hpp"
+#include "occa.hpp"
+#include "mm.hpp"
+#include "device.hpp"
+
+#ifdef MFEM_USE_RAJA
+#include "RAJA/RAJA.hpp"
+#endif
+
+namespace mfem
+{
+
+// OKINA = Okina Kernel Interface for Numerical Analysis
```

Comment:
Looks like a recursive name for Okina...

Reply:
I like the name! Reminds me of GNU.